src/UI/Implementation/Component/Input/Group.php

```diff
@@ -121,9 +121,11 @@ protected function nameInputs(NameSource $source, string $parent_name): array
     }
 
     /**
-     * @inheritdoc
+     * ATTENTION: This is not the same as isClientSideValueOk from the base class,
```

Comment:
Could we make this an annotation?
```suggestion
     * ATTENTION: This is not the same as @see Input::isClientSideValueOk() from the base class,
```